### PR TITLE
Cuira 345

### PR DESCRIPTION
**Pull request by sonny-hmcts**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/CUIRA-345

### Change description ###

added crossbrowser tests that run the feature tests

### Checklist

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

File: bin/run-crossbrowser-tests.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+export E2E_OUTPUT_DIR='./functional-output/crossbrowser'
+
+if [[ "$BROWSER_GROUP" == "" ]]
+then
+    EXIT_STATUS=0
+    BROWSER_GROUP=chrome yarn test-crossbrowser-e2e || EXIT_STATUS=$?
+    BROWSER_GROUP=firefox yarn test-crossbrowser-e2e || EXIT_STATUS=$?
+    BROWSER_GROUP=safari yarn test-crossbrowser-e2e || EXIT_STATUS=$?
+    BROWSER_GROUP=microsoft yarn test-crossbrowser-e2e || EXIT_STATUS=$?
+    echo EXIT_STATUS: $EXIT_STATUS
```
Comment (rapolaskaseliscgi) [inline bin/run-crossbrowser-tests.sh:14]: This will never fail, is this expected?

File: src/test/crossbrowser/SauceLabsReportingHelper.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+const event = require('codeceptjs').event;
+const container = require('codeceptjs').container;
+const exec = require('child_process').exec;
+
+const sauceUsername = process.env.SAUCE_USERNAME;
+const sauceKey = process.env.SAUCE_ACCESS_KEY;
+
+function updateSauceLabsResult(result, sessionId) {
+  console.log('SauceOnDemandSessionID=' + sessionId + ' job-name=cui-ra');
+  // eslint-disable-next-line max-len
```
Comment (rapolaskaseliscgi) [inline src/test/crossbrowser/SauceLabsReportingHelper.js:10]: Is this needed given how return statement is split over ~10 lines?

File: src/test/crossbrowser/supportedBrowsers.js [hunk 3]
```diff
@@ -0,0 +1,77 @@
+const LATEST_MAC = 'macOS 10.15';
```
Comment (rapolaskaseliscgi) [inline src/test/crossbrowser/supportedBrowsers.js:1]: That is fairly old MacOS version.. like 5 years old

File: src/test/crossbrowser/supportedBrowsers.js [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+const LATEST_WINDOWS = 'Windows 10';
```
Comment (rapolaskaseliscgi) [inline src/test/crossbrowser/supportedBrowsers.js:2]: This is not the latest, should we also add Win 11?

**Review by rapolaskaseliscgi — APPROVED** [review]

**Review by Ferdi-Cocelli — APPROVED** [review]

**Pull request merged**